packages/contracts/src/networks.ts

```diff
@@ -1269,6 +1269,152 @@ export const SPHINX_NETWORKS: Array<SupportedNetwork> = [
     hardcodedMerkleLeafGas: (11200000).toString(),
     actionTransactionBatching: false,
   },
+  {
+    name: 'mantle_sepolia',
+    displayName: 'Mantle Sepolia',
+    chainId: BigInt(5003),
+    rpcUrl: () => process.env.MANTLE_SEPOLIA_URL!,
+    rpcUrlId: 'MANTLE_SEPOLIA_URL',
+    blockexplorers: {
+      blockscout: {
+        browserURL: 'https://explorer.sepolia.mantle.xyz/',
+        apiURL: 'https://explorer.sepolia.mantle.xyz/api',
+        envKey: 'MANTLE_SEPOLIA_BLOCKSCOUT_API_KEY',
+        selfHosted: false,
+      },
+    },
+    currency: 'MNT',
+    dripSize: '1',
+    networkType: 'Testnet',
+    dripVersion: 0,
+    decimals: 18,
+    queryFilterBlockLimit: 2000,
+    legacyTx: false,
+    actionGasLimitBuffer: false,
+    eip2028: true,
+    actionTransactionBatching: true,
+  },
+  {
+    name: 'mantle',
+    displayName: 'Mantle',
+    chainId: BigInt(5000),
+    rpcUrl: () => process.env.MANTLE_MAINNET_URL!,
+    rpcUrlId: 'MANTLE_MAINNET_URL',
+    blockexplorers: {
+      blockscout: {
+        browserURL: 'https://explorer.mantle.xyz/',
+        apiURL: 'https://explorer.mantle.xyz/api',
+        envKey: 'MANTLE_BLOCKSCOUT_API_KEY',
+        selfHosted: false,
+      },
+    },
+    currency: 'MNT',
+    dripSize: '1',
+    networkType: 'Mainnet',
+    dripVersion: 0,
+    decimals: 18,
+    queryFilterBlockLimit: 2000,
+    legacyTx: false,
+    actionGasLimitBuffer: false,
+    eip2028: true,
+    actionTransactionBatching: true,
+  },
+  {
+    name: 'astar_zkyoto',
+    displayName: 'Astar zkEVM zkyoto',
```

Comment:
I think this should be consistent with the name in the main readme, i.e. "Astar zKyoto"